CONTRIBUTING.md

```diff
@@ -136,140 +155,169 @@ The process described here has several goals:
 
 Please follow these steps to have your contribution considered by the maintainers:
 
-1. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
+1. Follow all instructions in [Pull Request template](./Pull_Request_Template.md)
```

Comment:
As for the issue and bug templates, I'd be for letting people discover the template when then try to create a PR.